gammapy/background/reflected.py

```diff
@@ -5,24 +5,69 @@
 import numpy as np
 from numpy import sin, cos, arctan2, sqrt
 from ..image import exclusion_distance, lookup
+from astropy.table import Table
 
 __all__ = [
     'ReflectedRegionMaker',
-    'ReflectedBgMaker',
+    'CircularOffRegions',
+    'find_reflected_regions_for_event_list'
 ]
 
 
+class CircularOffRegions(Table):
```

Comment:
Now that I see it, I'm pretty sure that a table subclass is not a good solution here.

A `RegionList` class that holds circle region members would be much better abstraction.
For now we could use the photutils aperture classes to represent the circles in sky or pixel coordinates: http://photutils.readthedocs.org/en/latest/photutils/aperture.html#class-inheritance-diagram

But as I said ... if what you have now works maybe just move on to produce results and redo regions at some point in the future?